src/run-r.ts

```diff
@@ -0,0 +1,105 @@
+import * as vscode from "vscode";
+import { openBrowser, openBrowserWhenReady } from "./net-utils";
+import {
+  envVarsForShell as envVarsForTerminal,
+  escapeCommandForTerminal,
+} from "./shell-utils";
+import { getAppPort } from "./port-settings";
+
+export async function runApp(): Promise<void> {
```

Comment:
Could this have gone into the same module as the Python versions? No need to copy/paste the supporting functions `createTerminalAndCloseOthersWithSameName` and `getActiveEditorFile`?

Reply:
Good point, I wasn't sure how much would be the same between the two. Happy to refactor the R portion back into a single `run` module.

Reply:
I refactored to pull `rRunApp()` back into `run.ts` and I renamed the existing python functions to use a `py` prefix, e.g. `pyRunApp()` and `pyDebugApp()`